packages/cli-platform-ios/src/index.ts

```diff
@@ -4,3 +4,4 @@
 
 export {default as commands} from './commands';
 export {projectConfig, dependencyConfig, findPodfilePaths} from './config';
+export {runSudo} from './tools/installPods';
```

Comment:
`runSudo` is now a public API of `cli-platform-ios`. Is it obvious what command it runs sudo for? Not really. Let's rename it accordingly, `installPodsWithSudo` or something else relevant

Reply:
basically it takes a command as a parameter, so it can be used to run anything. Maybe it's better idea to move it to `cli-tools` then?

Reply:
makes more sense